src/viewProviders/resources.ts

```diff
@@ -63,13 +66,8 @@ export class ResourceViewProvider implements vscode.TreeDataProvider<ResourceVie
     this.treeView = vscode.window.createTreeView("confluent-resources", { treeDataProvider: this });
 
     ccloudConnected.event((connected: boolean) => {
+      logger.debug("ccloudConnected event fired", { connected });
       this.refresh();
-      // toggle Topics/Schemas views' visibility based on connection status
```

Comment:
The problem with this placement was that we would only set it during a `ccloudConnected` change event from the auth provider. 

Context: The main reason we pushed `ccloudConnected` deep into multiple condition checks in the auth provider's `getSessions()` was so we wouldn't end up calling the view provider's `.refresh()` over and over in the different views. We can safely move the `setContext` command into a more-frequently-called spot since it won't cause the UI to update unless the value actually changes.